src/meta/rule.schema.json

```diff
@@ -49,13 +49,20 @@
             "type": "string"
         },
         "severity": {
-            "enum": [
-                0,
-                1,
-                2,
-                3
-            ],
-            "type": "number"
+            "oneOf": [
+                {
+                    "enum": [
+                        0,
```

Comment:
Technically I guess it can be `-1` for `off`?

Reply:
Correct. I will add it.